environment.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
  - vispy
  - pip=19.1.1
  - pip:
-   - construct==2.5.1
+   - construct==2.9.45
```

Comment:
We shouldn't pin this if we don't have to.
Will this not work with 2.10.56 -- the latest?

Reply:
The PH5 web services are currently using 2.9.45, as I believe that was the latest until recently, so I want to make them match. I haven't looked at 2.10.56 yet.

Reply:
oh sure, I'm saying if it works without the pin leave it out and let the latest or what ever else may need get used.
If it's not backwards compatible add a minimum version. Ideally best to keep them as general as possible.

Reply:
It looks like version 2.10 drops support for python 2.7

https://construct.readthedocs.io/en/latest/transition210.html

We'll want to pin it to the latest version < 2.10